packages/remix-server-runtime/utils.ts

```diff
@@ -0,0 +1 @@
+export type PromiseOrSync<T> = Promise<T> | T;
```

Comment:
Is there a better place for `PromiseOrSync`?

Reply:
It's slightly different from the [`Promisable`](https://github.com/sindresorhus/type-fest/blob/HEAD/source/promisable.d.ts) type of @sindresorhus' `type-fest` package, but I guess the end-result is actually the same?

Should we use that one instead? 🤔